### DISCUSSION TITLE
source build of casadi using pip in windows 10

### DISCUSSION THREAD

**Discussion by sandeep026** [casadi/casadi#3415] [Q&A] [answered]
I want to test openmp support for map which requires source build.

There has been instruction provided for linux on the wiki to do a source build using pip.  I wanted to do that on my windows laptop and followed the following steps.

windows 10
msys2 mingw64
vscode

1. installed gfortran using msys2 mingw 64

2. And did 
```
export CASADI_SETUP_CMAKE_ARGS="-DWITH_BUILD_IPOPT=ON;-DWITH_BUILD_REQUIRED=ON;-DWITH_IPOPT=ON;-DWITH_SPRAL=ON;-DWITH_BUILD_SPRAL=ON;-DWITH_PYTHON=ON;-DWITH_PYTHON3=ON;-DWITH_OPENMP=ON;-DWITH_THREAD=ON;-DWITH_THREAD_MINGW=ON"
pip install --verbose casadi
```

But it did not build from source as an example with openmp falled back to serial evaluation.

I am not sure if I did the steps correctly or these are the correct steps. Please guide me.

**Comment by echandler5956f** [comment]
I had this issue too. Is there any additional documentation on this install option?

**Comment by sandeep026** [comment, marked as answer]
Create a fork of the casadi repository and modify the env key in the yaml file to trigger github action. From generated tarball, install using pip.

More information, check the [link](https://github.com/casadi/casadi/wiki/FAQ%3A-how-can-I-generate-my-own-binaries-on-my-CasADi-fork%3F) for more information.
